test.py

```diff
@@ -91,7 +90,7 @@ def test_bundle_from_cmd():
 
     argv = [MODULE_FILE_PATH, '-a', author, '-e', email, '-d', description,
             '-h', homepage, '-l', _license, '-c', command_line_args, '-r', min_redis_version,
-            '-rl', min_redis_pack_version, '-ca', ','.join([cap['name'] for cap in MODULE_CAPABILITIES])]
+            '-rl', min_redis_pack_version, '-ca', ','.join([cap['name'] for cap in MODULE_CAPABILITIES]), '-o', BUNDLE_ZIP_FILE]
```

Comment:
please use BUNDLE_ZIP_FILE const also in the assert phrase that checks this.
In addition it could be nice to put a non default value to the BUNDLE_ZIP_FILE.